webboot/webboot.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Copyright 2019 the webboot Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
```

Comment:
we had a discussion about this, we decided to leave it as u-root Authors

Reply:
Done.